forwarding/src/test/kotlin/com/pkware/file/forwarding/ForwardingFileSystemTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.pkware.file.forwarding
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import java.nio.file.FileSystem
+import java.nio.file.Paths
+
+class ForwardingFileSystemTest {
+
+    @Test
+    fun getPathVarargsAreCorrectlyForwarded() {
+        val mock = mock(FileSystem::class.java)
+        val stub = StubFileSystem(mock)
+
+        // Varargs with no entries
+        var path = Paths.get("first")
+        `when`(mock.getPath("first")).thenReturn(path)
```

Comment:
`mockito-kotlin` plays more nicely with Kotlin.

Reply:
these were tests originally made by marius and i am hesitant to modify them- is this a simple change?